src/Microsoft.AspNetCore.SignalR.Core/SignalRConnectionBuilderExtensions.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.AspNetCore.SignalR
     /// </summary>
     public static class SignalRConnectionBuilderExtensions
     {
+        /// <summary>
+        /// Configure a connection to map requests to the specified <see cref="Hub"/> type.
```

Comment:
At the connection level, we don't really have "requests". How about "Configure the connection to host the specified [Hub]" ?